sway/commands/titlebar_border_thickness.c

```diff
@@ -0,0 +1,30 @@
+#include <string.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/output.h"
+#include "sway/tree/arrange.h"
+#include "log.h"
+
+struct cmd_results *cmd_titlebar_border_thickness(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "titlebar_border_thickness", EXPECTED_EQUAL_TO, 1))) {
+		return error;
+	}
+
+	char *inv;
+	int value = strtol(argv[0], &inv, 10);
+	if (*inv != '\0' || value < 0 || value > config->titlebar_v_padding) {
+		return cmd_results_new(CMD_FAILURE, "titlebar_border_thickness",
+			"Invalid size specified");
+	}
+
+	config->titlebar_border_thickness = value;
+
+	for (int i = 0; i < root->outputs->length; ++i) {
+		struct sway_output *output = root->outputs->items[i];
+		arrange_workspace(output_get_active_workspace(output));
```

Comment:
Don't we need to use `config->titlebar_border_thickness` when arranging containers?

Reply:
Changing the border thickness and the padding should move containers on screen, right?

Reply:
containers stay the same, the size of the inner window (application) changes

Reply:
Containers themselves also change when you have nested tabbed/stacked containers.

In any case, `arrange_*` should be updated. We're probably hardcoding values in there.

Reply:
I see ...

I'll look for hardcoded values in arrange functions (the values I removed were not used over there) and try to see if arranging uses any value for the titlebar size ... but since the height of titlebars depend on the size of the font I don't know how an hardcoded value could be used
 
by the way, I've set up my environment with nested tabbed and split containers to visualize what happens, the rendering looks fine ...

Reply:
Just checked, `container_titlebar_height` is used. It uses your new config field. It seems `titlebar_border_thickness` is included in the padding. Looks good!